src/connector/postgres.rs

```diff
@@ -510,6 +515,11 @@ impl PostgreSql {
         })
     }
 
+    #[cfg(feature = "expose-drivers")]
```

Comment:
Document this? Or wait until we are sure we want this API?

Reply:
No. Reason about the need in rustdoc here to remind us later about the use.